.changeset/rich-birds-reply.md

```diff
@@ -0,0 +1,24 @@
+---
+'example-backend': minor
```

Comment:
No need to include this in the changeset.

Reply:
Roger. I think it was added by the `yarn changeset` command? Will remove ofc 👍